quantipy/core/quantify/engine.py

```diff
@@ -1989,6 +1997,12 @@ def _convert_statistic(self, teststat):
             dof = ebases_pairs - self.overlap - 2
             dof[dof <= 1] = np.NaN
             return get_pval(dof, teststat)[1]
+        elif self.mimic == 'LINK_legacy':
+            ebases_pairs = [eb1 + eb2 for eb1, eb2
+                            in combinations(self.ebases[0], 2)]
+            dof = ebases_pairs - self.overlap - 2
+            dof[dof <= 1] = np.NaN
+            return get_pval(dof, teststat)[1]
```

Comment:
**question:** Given that this is identical to Dimensions for now, shall we put this onto one together with it in the `if-elif-else`? Saying that, we might need to change this bit as remarked in the original ticket (this is a piece of part that might need to be changed after the review, if we find more not matching results). Probably better to leave it as you did here for the time being anyway!